CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - [#2113](https://github.com/wasmerio/wasmer/pull/2113) Bump minimum supported Rust version to 1.49
 
 ### Fixed
+- [#2125](https://github.com/wasmerio/wasmer/pull/2125) Fix RUSTSEC-2021-0023; `rand_core`: incorrect check on buffer length when seeding RNGs.
```

Comment:
This doesn't affect us, might want to mention that in the changelog, or remove this from the changelog entirely.

Reply:
I can remove it from the changelog, yes.